components/plans/create/AttributePicker.tsx

```diff
@@ -0,0 +1,231 @@
+import { PlusIcon, XMarkIcon } from "@heroicons/react/20/solid";
+import clsx from "clsx";
+import { useState } from "react";
+import { useMap } from "react-map-gl";
+import Button from "../../Button";
+import CardSlider from "../../CardSlider";
+
+interface AttributesPickerProps {
+  attributes: string[] | undefined;
+  selectedAttributes: string[][];
+  setSelectedAttributes: (stops: string[][]) => void;
+  attributesIndex: number;
+  setAttributesIndex: (index: number) => void;
+  setSelectedTabIndex: (index: number) => void;
+  suggestions: Venue[] | undefined;
```

Comment:
Quite a lot of shared state and likely prop drilling, could we consider using a context here something like AttributesContext to wrap whatever is needed?